### PR TITLE
Circuit output endianness and import path

### PR DESCRIPTION
**Pull request by danzipie**
Circuit outputs big-endian and imports circomlib

### CODE REVIEW

File: circuits/sha1.circom [hunk 1]
```diff
@@ -67,6 +68,12 @@ template Sha1(nBits) {
     }
 
     for (k=0; k<160; k++) {
-        out[k] <== sha1compression[nBlocks-1].out[k];
```
Comment (dmpierre) [inline circuits/sha1.circom:70]: I would rather have it written this way:

```
    for (i=0; i<5; i++) {
        for (k = 0; k < 32; k++) {
            out[32*i + k] <== sha1compression[nBlocks-1].out[32*i + (31-k)];
        }
    }
```

File: circuits/sha1.circom [hunk 2]
```diff
@@ -11,6 +11,7 @@ template Sha1(nBits) {
     var k;
     var nBlocks;
     var bitsLastBlock;
+    var v[160];
```
Comment (dmpierre) [inline circuits/sha1.circom:14]: You can remove it, as it shouldn't be needed

File: circuits/sha1.circom [hunk 3]
```diff
@@ -67,6 +68,12 @@ template Sha1(nBits) {
     }
 
     for (k=0; k<160; k++) {
-        out[k] <== sha1compression[nBlocks-1].out[k];
+        v[k] = sha1compression[nBlocks-1].out[k];
+    }
+
+    for (i=0; i<160/32; i++) {
```
Comment (dmpierre) [inline circuits/sha1.circom:74]: Similarly, you can remove this part

**Review by dmpierre — CHANGES_REQUESTED** [review]
Hey thanks for this! A few suggestions on what could be changed within your PR

File: circuits/sha1.circom (hunk 1 above)
Comment (danzipie) [reply circuits/sha1.circom:70]: Changed as requested

**Review by danzipie — COMMENTED** [review]
Changed as requested

**Pull request merged**